ember/app/components/welcome-modal.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+import { action, computed } from 'ember-decorators/object';
+import { service } from 'ember-decorators/service';
+
+export default class extends Component {
+
+  @service cookies
+  constructor() {
+    super();
+    this.classNameBindings = ['getClassNames'];
+    this.active = !this.get('cookies').read()['dismissed_welcome'];
+  }
+
+  @computed('active')
+  getClassNames() {
+    return this.get('active')
+        ? 'component welcome-modal active'
+        : 'component welcome-modal';
```

Comment:
I would do a conditional render since it's not something that needs to change on the fly. It's either there or it isn't, and almost all of the time it isn't.

Reply:
I'm pretty sure a className change would trigger a rerender anyway so I don't think that it would be any less performant to conditionally render.